content/blog/2022/06/2022-06-13-Let-the-coding-start.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Let the (GSoC) codeing start !"
+tags:
+- gsoc
+- gsoc2022
+- events
+- community
+authors:
+- jmMeessen
+sig: 
+- gsoc
+description: >
+  GSoC Bonding phase is completed. The Summer of Coding for the four Jenkins projects starts this Monday!" 
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+The GSoC "Bonding Period" just finished. 
+
+The four Jenkins GSoC projects are now ready. 
+Contact between Contributors and Mentors is established.
```

Comment:
Lower case rather than upper case, not an English language "proper noun".
```suggestion
Contact between contributors and mentors is established.
```